src/index.js

```diff
@@ -0,0 +1,54 @@
+const cluster = require ("cluster");
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9973834)
```suggestion
const cluster = require("cluster");
```